README.md

```diff
@@ -728,6 +732,18 @@ Then, update your config to enable SSR (if you'd like to enable it globally).
     raise_on_ssr_failure: config_env() != :prod
 ```
 
+### Using ESM (EcmaScript Modules) on SSR entrypoint
+
+By default this library uses CommonJS modules for SSR. If you want to use ESM (EcmaScript Modules) set `esm: true` in your config.
+
+```elixir
+  {Inertia.SSR, path: Path.join([Application.app_dir(:my_app), "priv"]), esm: true},
+```
+
+### Custom SSR adapter
+
+You can setup your own SSR adapter. For example [vitex](https://github.com/andresgutgon/vitex) is a package that helps with ViteJS development in Phoenix apps. It has a custom SSR adapter for this package (inertia-phoenix) so SSR can be handle in development through Vite Dev server instead of calling a NodeJS process like we do in production.
```

Comment:
@derrickreimer you commented that maybe would be better if adapters lived outside the Inertia package. And I think I really like this approach because I want to add Vite assets management and that would be too much to put in inertia.